persist/fs/commitlog/commit_log.go

```diff
@@ -50,8 +50,10 @@ type commitLog struct {
 	sync.RWMutex
 	opts    Options
 	nowFn   clock.NowFn
-	metrics xmetrics.Scope
-	log     xlog.Logger
+	scope   tally.Scope
+	metrics *commitLogMetrics
```

Comment:
Perhaps just make this a struct value rather than a ref?  Slightly better memory locality.